samples/custom_dispatch/cpu/embedded/functions.c

```diff
@@ -85,3 +85,33 @@ void simple_mul_inplace_workgroup(
     binding1[i] *= binding0[i];
   }
 }
+
+// `ret = -|lhs * rhs|`
+//
+// Conforms to ABI:
+// #hal.pipeline.layout<push_constants = 1, sets = [
+//   <0, bindings = [
+//       <0, storage_buffer, ReadOnly>,
+//       <1, storage_buffer, ReadOnly>,
+//       <2, storage_buffer>
+//   ]>
+// ]>
+// With a workgroup size of 64x1x1.
+void simple_mul_abs_negate_workgroup(
```

Comment:
I am not sure how this gets converted to the object file and embedded into the final vmfb. I know its part of other samples, which I havent seen ,but thats interesting.